exporter/opentelemetry-exporter-zipkin-json/src/opentelemetry/exporter/zipkin/json/__init__.py

```diff
@@ -119,7 +138,22 @@ def __init__(
         elif version == Protocol.V2:
             self.encoder = JsonV2Encoder(max_tag_value_length)
 
+        self.session = requests.Session()
+        self.session.headers.update(
+            {"Content-Type": self.encoder.content_type()}
+        )
+        self.done = False
```

Comment:
may be rename to `_closed`? done is a bit ambiguous. At first glance it is not clear whether this refers to a single batch or entire instance.

Reply:
Sounds good, will do.